src/main/java/views/LeagueSimulatorPane.java

```diff
@@ -0,0 +1,296 @@
+package views;
+
+import controllers.SceneManager;
+import javafx.application.Platform;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+
+import java.lang.reflect.Array;
+import java.util.*;
+
+import javafx.scene.text.Text;
+import models.*;
+
+
+public class LeagueSimulatorPane extends GamePane {
+
+    Game GAME = Game.getInstance();
+
+    private final VBox leftContainer = new VBox();
+    private final VBox matchesContainer = new VBox();
+    private final VBox centerContainer = new VBox();
+
+    private Label title = new Label("League Simulator");
+    private final Button returnButton = new Button("Return");
+    private final Button playButton = new Button("Start");
+    private final Button refreshButton = new Button("Clear");
+
+    private Label sizeLabel = new Label("League size:");
+    final ComboBox<String> size = new ComboBox<>();
+    private HBox sizeBox = new HBox();
+
+    ArrayList<String > teamNames = GAME.getAllTeamNames();
+
+    private Label match1 = new Label("Match 1:");
```

Comment:
I feel like this should be scaleable with the several types of boxes in an array, this would result in cleaner code further down in the code too. Lots of code being repeated here!